tools/realsense-viewer/CMakeLists.txt

```diff
@@ -109,12 +109,12 @@ endif()
 
 # config-ui
 if(WIN32)
-    add_executable(realsense-viewer WIN32
+    add_executable(realsense-viewer 
```

Comment:
The WIN32 tag must also be reverted, otherwise the console will be visible

Reply:
I was wondering about that... thanks!

Reply:
Fixed